src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
You need to split the logic into three private methods and call them one by one in the public method 
1) only reads data and returns strings
2) accepts lines, calculates and forms the text of the report
3) accepts the report and only writes it to a file